pdns/bpf-filter.cc

```diff
@@ -602,6 +740,32 @@ std::vector<std::pair<ComboAddress, uint64_t> > BPFFilter::getAddrStats()
     }
   }
 
+  {
```

Comment:
There is a small issue here because we cannot return the mask, so perhaps it would be better to implement a new `std::vector<std::pair<Netmask, uint64_t> > BPFFilter::getRangeStats()` (or something like that) method instead?